starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,63 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a
+  } else if (a < b) {
+    return b
+  } else {
+    return a;
+  }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(words) {
+  let result = "";
+  if (words.length === 0) {
+    return null
+  }
+  for (let i = 0; i < words.length; i++) {
+
+    if (result.length <= words[i].length) {
+      result = words[i];
+    }
```

Comment:
para el apartado de sí dos palabras tienen la misma longitud devolver la primera, como os dije ayer casi lo teníais.
Lo único que os sobra es el = del if ya que si es igual te va a meter la segunda palabra que le llega y vosotros queréis que se quede la que está por lo que tenéis que comprobar solo si la longitud es mayor no mayor e igual